scale-testing/scale.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+
+import subprocess, os
+from subprocess import Popen, PIPE
+import time
+import json
+import test_request
+
+
+"""
+THIS IS A DESCRIPTION OF WHAT THIS CLASS DOES
+"""
+
+
+class scale:
+
+    runtime = 0
+
```

Comment:
Remove extra line breaks.  In general be more consistent about line breaks.  Since this is a new file, try using the python formatter on all the files.